magpie/__init__.py

```diff
@@ -36,18 +36,19 @@ def __init__(self, keras_model=None, word2vec_model=None, scaler=None,
             self.scaler = scaler
 
     def train(self, train_dir, vocabulary, test_dir=None, callbacks=None,
-              nn_model=NN_ARCHITECTURE, batch_size=BATCH_SIZE,
+              nn_model=NN_ARCHITECTURE, batch_size=BATCH_SIZE, test_split=0.0,
```

Comment:
If possible yes, and maybe explain in the documentation exactly what you wrote me :+1: 

Newbie question: how can you train without testing?